lib/internal/streams/destroy.js

```diff
@@ -42,7 +51,10 @@ function destroy(err, cb) {
   // With duplex streams we use the writable side for state.
   const s = w || r;
 
-  if (w?.destroyed || r?.destroyed) {
+  if (
+    (w && (w[kState] ? (w[kState] & kDestroyed) !== 0 : w.destroyed)) ||
+    (r && (r[kState] ? (r[kState] & kDestroyed) !== 0 : r.destroyed))
```

Comment:
Why did you use the default behavior of accessing `destroyed` while in the rest you did not?

Reply:
Actualy. here it's not necessary. I will cahnge it.